server/entity/splashable.go

```diff
@@ -18,6 +19,18 @@ type splashable struct {
 	t potion.Potion
 }
 
+// SplashableBlock is a block that can be splashed with a splash bottle.
+type SplashableBlock interface {
+	world.Block
+	Splash(e world.Entity, w *world.World, pos cube.Pos, p potion.Potion)
```

Comment:
Could maybe do with some documentation on this function.

Reply:
Also, is the Splashable entity passed even used anywhere? If not, you can probably remove it here and in SplashableEntity.